go/cmd/deploy/main.go

```diff
@@ -106,22 +104,21 @@ var DeployFlags = []cli.Flag{
 	cli.Bool("init", "Initialize configuration file in the specified directory"),
 	cli.Bool("force", "Force overwrite existing configuration file when using --init"),
 	// Required flags (can be provided via config file)
-	cli.String("workspace-id", "Workspace ID", cli.EnvVar(EnvWorkspaceID)),
 	cli.String("project-id", "Project ID", cli.EnvVar("UNKEY_PROJECT_ID")),
-	cli.String("keyspace-id", "Keyspace ID for API key authentication", cli.EnvVar(EnvKeyspaceID)),
+	cli.String("api-id", "API ID for API key authentication", cli.EnvVar(EnvApiID)),
 	// Optional flags with defaults
-	cli.String("context", "Build context path"),
+	cli.String("context", "Build context path", cli.Default(".")),
 	cli.String("branch", "Git branch", cli.Default(DefaultBranch)),
 	cli.String("docker-image", "Pre-built docker image"),
 	cli.String("dockerfile", "Path to Dockerfile", cli.Default(DefaultDockerfile)),
 	cli.String("commit", "Git commit SHA"),
 	cli.String("registry", "Container registry",
 		cli.Default(DefaultRegistry),
 		cli.EnvVar(EnvRegistry)),
-	cli.String("env", "Environment slug to deploy to", cli.Default("preview")),
+	cli.String("env", "Environment slug to deploy to", cli.Default(DefaultEnvironment)),
 	cli.Bool("skip-push", "Skip pushing to registry (for local testing)"),
 	cli.Bool("verbose", "Show detailed output for build and deployment operations"),
-	cli.Bool("linux", "Build Docker image for linux/amd64 platform (for deployment to cloud clusters)"),
+	cli.Bool("linux", "Build Docker image for linux/amd64 platform (for deployment to cloud clusters)", cli.Default(true)),
```

Comment:
let's remember to announce this in slack, it's going to break people's local setup and produces a really hard to debug error sometimes